.env.example

```
@@ -2,3 +2,8 @@
 RESEND_API_KEY=re_123456789
 RESEND_FROM=onboarding@resend.dev
 SUPPORT_EMAIL=ftc@bd73.com
+
+# Slack Integration (optional — required for Slack notification channel)
+SLACK_CLIENT_ID=           # Slack app OAuth client ID
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)